rotkehlchen/db/dbhandler.py

```diff
@@ -583,6 +581,52 @@ def set_settings(self, write_cursor: 'DBCursor', settings: ModifiableDBSettings)
         )
         CachedSettings().update_entries(settings)
 
+    @overload
+    def get_cache(self, cursor: 'DBCursor', name: Literal[
+        DBCache.LAST_WRITE_TS,
+        DBCache.LAST_BALANCE_SAVE,
+        DBCache.LAST_DATA_UPLOAD_TS,
+        DBCache.LAST_DATA_UPDATES_TS,
+        DBCache.LAST_OWNED_ASSETS_UPDATE,
+        DBCache.LAST_EVM_ACCOUNTS_DETECT_TS,
+        DBCache.LAST_SPAM_ASSETS_DETECT_KEY,
+        DBCache.LAST_AUGMENTED_SPAM_ASSETS_DETECT_KEY,
+    ]) -> Timestamp:
+        ...
+
+    @overload
+    def get_cache(self, cursor: 'DBCursor', name: Literal[
+        DBCache.SPAM_ASSETS_VERSION,
+        DBCache.RPC_NODES_VERSION,
+        DBCache.CONTRACTS_VERSION,
+        DBCache.GLOBAL_ADDRESSBOOK_VERSION,
+        DBCache.ACCOUNTING_RULES_VERSION,
+    ]) -> int:
+        ...
+
+    @overload
+    def get_cache(self, cursor: 'DBCursor') -> dict[DBCache, DBCacheFieldTypes]:
+        ...
+
+    def get_cache(self, cursor: 'DBCursor', name: DBCache | None = None) -> dict[DBCache, DBCacheFieldTypes] | DBCacheFieldTypes:  # noqa: E501
+        """Returns the cache values from the `key_value_cache` table of the DB
+        If the given name doesn't exist in the table, returns 0 for timestamps, otherwise None"""
```

Comment:
Waht do you mean if the given name does not exist in the table? Don't you mean if there is nothing passed?

Reply:
Updated the comment to make it more clear:
```
Returns the cache values from the `key_value_cache` table of the DB
- If `name` is not passed, it returns all the values in the table
- If `name` is passed, it returns the value of the given name, defaults to Timestamp(0)
````